src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -428,5 +429,46 @@ private enum TimeSuffix
 			Ms,
 			S
 		}
+
+
+		protected string ParseCaptureBody(ConfigurationKeyValue kv)
+		{
+			var captureBodyInConfig = kv.Value;
+			if (string.IsNullOrEmpty(captureBodyInConfig))
+			{
```

Comment:
codestyle: the `{` `}` can be removed here.

Reply:
Done.